sqlest/src/test/scala/sqlest/sql/DB2StatementBuilderSpec.scala

```diff
@@ -235,6 +235,19 @@ class DB2StatementBuilderSpec extends BaseStatementBuilderSpec {
     )
   }
 
+  it should "produce the right SQL for nullable literal columns" in {
+    sql {
+      select(TableThree.col3, TableThree.col4, Option.empty[String].constant.as("testOption"))
```

Comment:
The compilation failed for `Option.empty[String].as("testOption")` with `value as is not a member of Option[String]`.

Just using `Option.empty[String]` I got the scarier error
```
[error]   [C1, C2, C3](column1: sqlest.ast.AliasedColumn[C1], column2: sqlest.ast.AliasedColumn[C2], column3: sqlest.ast.AliasedColumn[C3])sqlest.ast.syntax.SelectBuilder[(sqlest.ast.AliasedColumn[C1], sqlest.ast.AliasedColumn[C2], sqlest.ast.AliasedColumn[C3])]
[error]  cannot be applied to (sqlest.ast.TableColumn[Option[String]], sqlest.ast.TableColumn[Option[String]], Option[String])
```
In my original problem case the empty option does seem to end up as a constant so I think this will still be recreating that? It is basically there to make it compile though so I'm open to better ideas.